packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -369,13 +369,16 @@ abstract contract CrossDomainMessenger is
      *
      * @return Amount of gas required to guarantee message receipt.
      */
-    function baseGas(bytes calldata _message, uint32 _minGasLimit) public pure returns (uint32) {
+    function baseGas(bytes calldata _message, uint32 _minGasLimit) public pure returns (uint64) {
```

Comment:
I think you should make `_minGasLimit` a `uint64` as an argument here

Reply:
Decided on `uint32` based on reasoning [here](https://github.com/ethereum-optimism/optimism/pull/3542#issuecomment-1266331518). LMK what you think.